roles/timezone/tasks/main.yml

```diff
@@ -14,3 +14,16 @@
   community.general.timezone:
     hwclock: "{{ timezone_hwclock }}"
     name: "{{ timezone_name }}"
+
+- name: Create /etc/adjtime file if not exists
+  become: true
+  ansible.builtin.file:
+    path: /etc/adjtime
+    state: touch
+    mode: '0644'
+
+- name: Ensure UTC in /etc/adjtime
+  become: true
+  ansible.builtin.lineinfile:
+    path: /etc/adjtime
+    line: "{{ timezone_hwclock }}"
```

Comment:
Why was this added?

Reply:
Only for CentOS needed. changed tasks this way: 

```
- name: RedHat specific tasks
  become: true
  block:
    - name: Create /etc/adjtime file if not exists
      ansible.builtin.file:
        path: /etc/adjtime
        state: touch
        mode: '0644'
    - name: Ensure UTC in /etc/adjtime
      ansible.builtin.lineinfile:
        path: /etc/adjtime
        line: "{{ timezone_hwclock }}"
  when: "'RedHat' in ansible_os_family"
```